src/Cookie.php

```diff
@@ -89,6 +102,19 @@ public function __construct(
         $this->httpOnly = (bool) $httpOnly;
     }
 
+    public static function createWithoutValidation(
+        $name,
+        $value = null,
+        $maxAge = null,
+        $domain = null,
+        $path = null,
+        $secure = false,
+        $httpOnly = false,
+        \DateTime $expires = null
+    ) {
+        return new self($name, $value, $maxAge, $domain, $path, $secure, $httpOnly, $expires, false);
```

Comment:
I would do some hacks here: pass null or the valid empty value to bypass the validation, then set the property directly. Since we are in the same class, we can set private properties.